node/src/lib.rs

```diff
@@ -83,6 +87,7 @@ impl NodeStorage {
         ) = reopen!(&rocksdb,
             Self::HEADERS_CF;<HeaderDigest, Header>,
             Self::CERTIFICATES_CF;<CertificateDigest, Certificate>,
+            Self::CERTIFICATE_ID_BY_ROUND_CF;<(Round, CertificateDigest), u8>,
```

Comment:
What is the `u8` field?

Reply:
@asonnino nothing really - since it's a key-value store we need to have something on the value side. We use this table just for the ordering purposes of the keys by round. That tells me that I might need to use a type alias so to make things a bit clearer